### DISCUSSION TITLE
Question: Multilingual Namespacing

### DISCUSSION THREAD

**Discussion by heraldodinero** [requarks/wiki#7340] [Help / Questions]
Hello wiki.js folks! 

I've searched the questions and I don't see any on this topic already.. 

I have "Multilingual Namespacing" set to "Disabled" in the "Site" -> "Locale" configuration.

I've created some new pages to see if that makes any difference, but pages still automatically load to /en/page.

I have absolutely no use what-so-ever for multilingual features, and would preferably like to disable these if possible... The older version I used when I was introduced to wiki.js didn't have the languages being forced in the URL and page router.

For my uses, it would be much easier for me and my organization just to be able to turn this feature off, or as close as possible to off.

Fortunately, I see there is a config option to disable "Multilingual Namespacing", however I've tried toggling it off, and I still get /en/ (my sites default language) automatically added to all the URLs that get navigated to throughout the wiki.

I know that our users can just go directly to mywiki.domain/page, however, it would give a much cleaner presentation if they weren't being shown mywiki.domain/en/page, instead.

Is there any way to disable multi-lingual options?

**Comment by DarthJahus** [comment]
I'd like an answer as well.

**Comment by joshuasharpe-bnsf** [comment]
Related to: https://github.com/requarks/wiki/discussions/7458
